distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -49,16 +49,21 @@ javax.net.ssl.trustStoreType=jks
 #
 # Global URL Properties
 #
-# The httpsPort, httpPort, and port properties are used to advertise the system externally
```

Comment:
We have to change them either way. Since the existing properties had to match the system DDF was running on, those were really more internal than external already.

Reply:
❓ Why not keep the properties as is to mean as it did external and internal. And have an optional one for internal that when defined is used and if not, the normal one continues to be used. That way, the system continue to work as it did and yet we have the option to define an internal hostname when needed?

Reply:
Talked to @pklinef and he wasn't overly concerned about renaming the system properties. Those are not guaranteed to remain between releases so we're free to change them. It still bugs me a little though that we had a comment saying
```
The httpsPort, httpPort, and port properties are used to advertise the system externally
```
and we've changed their meaning to be internal now and added new `external` ones. Seems to me that keeping the old system property names for external ones and create new `internal` (or `instance`, or `node`) ones would have limited the impact, both for DDF and downstream projects.

Reply:
I think that comment was always wrong. They go with the system hostname, which is required to match the machine hostname, the certificate DA, and the keystore alias.

External might also be an overloaded term, which doesn't help matters.